deploy/charts/harvester/templates/configmap.yaml

```diff
@@ -64,3 +64,16 @@ data:
       sleep 2
     done
     `}}
+
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: installer-provided-settings
+  labels:
+{{ include "harvester.labels" . | indent 4 }}
+immutable: true
+data:
+{{- range $key, $value := .Values.service.installerProvidedSettings }}
```

Comment:
Is there a better alternative to the `installerProvided` term?
Like `default` or `preConfigured`.  The term `installer` is too specific.
I can load the chart and provide some default settings and the source doesn't have to be the installer.

Reply:
What about `userSettings`? This indicates that these are provided by the user.